requests/compat.py

```diff
@@ -37,7 +37,8 @@
 # ---------
 
 if is_py2:
-    from urllib import quote, unquote, quote_plus, unquote_plus, urlencode, getproxies, proxy_bypass
+    from urllib import quote, unquote, quote_plus, unquote_plus, urlencode, getproxies, proxy_bypass, \
```

Comment:
It may be better to follow something closer to the import convention of the other modules, rather than use `\`.
```python
from urllib import (
    quote, unquote, quote_plus, unquote_plus, urlencode, getproxies,
    proxy_bypass, proxy_bypass_environment, getproxies_environment)
```